### ISSUE TITLE
Thumbnail creator modal should be able to be closed with keyboard

### ISSUE THREAD

**Issue by davidschober** [avalonmediasystem/avalon#2525]
## Description

To improve accessibility, thumbnail creator modal should be able to be closed. 

## done looks like 

- the modal closes.
 
From #2130

**Comment by joncameron** [comment]
Worked in Firefox but not Chrome; other UI elements are available for possible accessibility purposes.
